pyanaconda/installclasses/fedora_server.py

```diff
@@ -1,5 +1,5 @@
 #
-# fedora-server.py
+# fedora_server.py
```

Comment:
Personally I would like to get rid of these file names as comments on top of a file. First of all, why we need them in a first place?

Reply:
Fixed.